test/TEST-03-JOBS/test-jobs.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -x
+#!/bin/bash -e
```

Comment:
`-x` has been here since the first version of the test was merged, so I'm not sure it should be removed. I think something like
```sh
#!/bin/bash
set -x
set -e
```
could be used instead.